credit/data.py

```diff
@@ -1443,6 +1492,316 @@ def __iter__(self):
                     break
 
 
+# class Predict_Dataset(torch.utils.data.IterableDataset):
```

Comment:
Before you merge, delete this old dataset and rename the new one Predict_Dataset.